src/test/java/com/github/onsdigital/dp/authorisation/permissions/CachingStoreTest.java

```diff
@@ -2,20 +2,22 @@
 
 
 import com.github.onsdigital.dp.authorisation.permissions.models.Bundle;
+import com.github.onsdigital.dp.authorisation.exceptions.BundleNotCached;
 import org.hamcrest.CoreMatchers;
 import org.joda.time.Duration;
 import org.junit.Before;
 import org.junit.Test;
 import org.mockito.Mock;
 import org.mockito.MockitoAnnotations;
 
+import static junit.framework.TestCase.assertTrue;
```

Comment:
The imports on lines 13 and 16 (TestCase.assertTrue and Matchers.eq respectively) are not being used so can also be deleted.